gnome-project/backtrack/views.py

```diff
@@ -46,16 +46,19 @@ def get_context_data(self, **kwargs):
 
             if len(sprint_list_current) != 0:
                 context["sprint_current"] = sprint_list_current[0]
-                sprint_current_id = sprint_list_current[0].id
+                '''sprint_current_id = sprint_list_current[0].id
```

Comment:
what is this

Reply:
That's a multiline comment. I preserved the old code just in case the new one had issues but I didn't remove the old one yet.